ddtrace/internal/sampling.py

```diff
@@ -208,44 +211,90 @@ def apply_span_sampling_tags(self, span):
 
 def get_span_sampling_rules():
     # type: () -> List[SpanSamplingRule]
-    json_rules_raw = os.getenv("DD_SPAN_SAMPLING_RULES")
-    if json_rules_raw is None:
+    json_rules = get_span_sampling_json()
```

Comment:
All but the `get_span_sampling_rules` function can be private so prefixed with underscore.